### PR TITLE
Runbook updates

### PR DESCRIPTION
**Pull request by xrstf**
**What this PR does / why we need it**:
This PR extends the alerting rules to provide more information for our documentation:

* Runbook URLs for all alerts.
* Remediation steps for some common alerts.
* Trailing newlines have been removed because they made the Slack alerts look ugly.

**Release note**:
```release-note
NONE
```

### CODE REVIEW

File: config/monitoring/prometheus/rules/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Prometheus Alerting/Recording Rules
+
+When editing these rules, please make sure to also update the
+runbook in the documentation repository by running
+`make runbook` in that repository.
```
Comment (kron4eg) [inline config/monitoring/prometheus/rules/README.md:5]: Is `make runbook` a take for follow-up PR? Because I can't see it there yet.

Comment (xrstf) [reply config/monitoring/prometheus/rules/README.md:5]: Yep

**Pull request merged**